src/Microsoft.AspNetCore.Razor.Language/DefaultRazorCodeGenerationOptionsBuilder.cs

```diff
@@ -11,9 +11,11 @@ internal class DefaultRazorCodeGenerationOptionsBuilder : RazorCodeGenerationOpt
 
         public override bool IndentWithTabs { get; set; }
 
+        public override bool GenerateChecksum { get; set; } = true;
```

Comment:
Defaulting this to `true` here. Should we instead have a `RuntimeOptionsFeature` like we do for design time? https://github.com/aspnet/Razor/blob/ajbaaska/nuke-checksum/src/Microsoft.AspNetCore.Razor.Language/DesignTimeOptionsFeature.cs#L30